back/config/schemas/settings.schema.json.erb

```diff
@@ -1030,6 +1030,18 @@
           "allowed": { "type": "boolean", "default": false },
           "enabled": { "type": "boolean", "default": false }
         }
+      },
+      
+      "analytics": {
+        "type": "object",
+        "title": "Analytics",
+        "description":  "Provides universal API for platform data and new insights charts.",
+        "additionalProperties": false,
+        "required": ["allowed", "enabled"],
+        "properties": {
+          "allowed": { "type": "boolean", "default": false },
+          "enabled": { "type": "boolean", "default": false }
+        }
```

Comment:
Engines add their feature flags dynamically using `CitizenLab::Mixins::FeatureSpecification`.

See https://www.notion.so/citizenlab/How-to-extend-settings-with-new-features-c948e0b84e4445bbb3b30eb875e125c3 for the documentation and for examples, you can have a look at other engines. I can also help if that's still not clear :)

Reply:
ok i remove that part that i had added manually and updated the feature specification file but still don't know if i have to update this file somehow.

Reply:
No, you don't need to modify `settings.schema.json.erb` in this case.